include/tvm/relay/transform.h

```diff
@@ -182,6 +183,122 @@ class Pass : public NodeRef {
   using ContainerType = PassNode;
 };
 
+class Sequential;
+
+/*!
+ * \brief The SequentialNode contains a set of passes that transform Relay
+ * programs from one AST to another semantically equivalent one.
+ *
+ * One example of this level of pass is that the pass manager needs to correctly
+ * perform a host of optimizations with a given optimization level and disabled
+ * passes.
+ */
+class SequentialNode : public PassNode {
```

Comment:
declare node, but hide the implementation inside. (User don't need to see SequentialNode, they only need to see Sequantial)